### PR TITLE
v2: Added jumpboxes host group and winrm config

### PR DESCRIPTION
**Pull request by suhanselvan**
Changes in this PR: 
-> Added Terraform config for WinRM
-> Added jumpboxes host group to ansible inventory file.
-> Added SHINE component which will be installed as a part of this sprint and removed Saprouter component which is not going to be installed as a part of this sprint.

### CODE REVIEW

File: deploy/v2/terraform/modules/jumpbox/winrm_files/FirstLogonCommands.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<FirstLogonCommands>
+    <SynchronousCommand>
+        <CommandLine>cmd /c "mkdir C:\terraform"</CommandLine>
+        <Description>Create the Terraform working directory</Description>
+        <Order>11</Order>
+    </SynchronousCommand>
+    <SynchronousCommand>
+        <CommandLine>cmd /c "copy C:\AzureData\CustomData.bin C:\terraform\winrm.ps1"</CommandLine>
```
Comment (AprilZhaoWY) [inline deploy/v2/terraform/modules/jumpbox/winrm_files/FirstLogonCommands.xml]: Can we guarantee the file is always in `C:\AzureDate\CustomData.bin`?

File: deploy/v2/template_samples/single_node_hana.json [hunk 2]
```diff
@@ -66,8 +66,7 @@
           "password": "Sap@hana2019!"
         },
         "components": [
-          "hana_studio_windows",
-          "sap_router"
```
Comment (AprilZhaoWY) [inline deploy/v2/template_samples/single_node_hana.json]: Please add to the description why the removal of `sap_router`

File: deploy/v2/template_samples/single_node_hana.json [hunk 3]
```diff
@@ -142,15 +140,16 @@
...
         "xs": [
           "xsac_alm_pi_ui",
           "xsac_portal_serv",
           "xsac_services",
           "xsac_ui5_fesv5",
           "xsac_ui5_sb",
           "xsac_xsa_cockpit"
-        ]
+        ],
+        "shine": []
```
Comment (AprilZhaoWY) [inline deploy/v2/template_samples/single_node_hana.json]: Please add to the description why the add of `shine`

File: deploy/v2/terraform/modules/jumpbox/winrm_files/FirstLogonCommands.xml (hunk 1 above)
Comment (suhanselvan) [reply deploy/v2/terraform/modules/jumpbox/winrm_files/FirstLogonCommands.xml]: Yes, That is the default path where the custom data files are stored.

https://azure.microsoft.com/is-is/blog/custom-data-and-cloud-init-on-windows-azure/

File: deploy/v2/template_samples/single_node_hana.json (hunk 2 above)
Comment (suhanselvan) [reply deploy/v2/template_samples/single_node_hana.json]: Added.

File: deploy/v2/template_samples/single_node_hana.json (hunk 3 above)
Comment (suhanselvan) [reply deploy/v2/template_samples/single_node_hana.json]: Added.

File: deploy/v2/template_samples/single_node_hana.json [hunk 4]
```diff
@@ -122,7 +120,7 @@
         "sku": "7.3"
       },
       "size": "Demo",
-      "filesystem": "XFS",
+      "filesystem": "xfs",
```
Comment (pabowers) [inline deploy/v2/template_samples/single_node_hana.json]: why the change in capitalization?

Comment (suhanselvan) [reply deploy/v2/template_samples/single_node_hana.json]: Apparently Ansible is case sensitive for file system fstype parameters. So changed it to lower case to avoid errrors.
